DQM/SiStripMonitorClient/plugins/SiStripAnalyser.cc

```diff
@@ -103,7 +101,8 @@ void SiStripAnalyser::analyze(edm::Event const& e, edm::EventSetup const& eSetup
       actionExecutor_.createDummyShiftReport();
     } else {
       auto& dqm_store = *edm::Service<DQMStore>{};
-      actionExecutor_.fillStatus(dqm_store, detCabling_, eSetup);
+      actionExecutor_.fillStatus(
+          dqm_store, detCabling_, &eSetup.getData(tkDetMapTokenELB_), &eSetup.getData(tTopoTokenELB_));
```

Comment:
The token was declared to be consumed in `EndLuminosityBlock`, but is begin used in `analyze`.